mobile/common.go

```diff
@@ -228,3 +228,17 @@ func (a *Addresses) Set(index int, address *Address) error {
 func (a *Addresses) Append(address *Address) {
 	a.addresses = append(a.addresses, address.address)
 }
+
+// ToHex returns the hex representation of b, prefixed with '0x'.
+// For empty slices, the return value is "0x0".
+//
+// Deprecated: use hexutil.Encode instead.
+func ToHex(b []byte) string {
+	return common.ToHex(b)
+}
+
+// FromHex returns the bytes represented by the hexadecimal string s.
+// s may be prefixed with "0x".
+func FromHex(s string) []byte {
+	return common.FromHex(s)
```

Comment:
Please change both of these functions to use package hexutil. FromHex should return an error.